src/mesido/electricity_physics_mixin.py

```diff
@@ -149,14 +174,22 @@ def path_variables(self):
         variables.extend(self.__asset_is_switched_on_var.values())
         variables.extend(self.__storage_charging_var.values())
         variables.extend(self.__set_point_var.values())
+        options = self.energy_system_options()
+        if options["electrolyzer_efficiency"] == ElectrolyzerOption.LINEARIZED_THREE_LINES_EQUALITY:
+            for line in self.__electrolyzer_is_active_linear_segment_var.keys():
+                variables.extend(self.__electrolyzer_is_active_linear_segment_var[line].values())
```

Comment:
in case updated with the map of __electrolyzer_is_active_linear_segment_var not having a nested dict, this loop can even be fully ommitted.

Reply:
Changed